Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs

```diff
@@ -223,10 +228,7 @@ public static void ConfigureProject(HashSet<Configurations> filter = null)
             {
```

Comment:
Do we know what scenarios filter will be null?

I don't think that we use this functionality right now (i.e. no code invokes with a null filter) but if this happens we'll have the same problem where the IsActiveBuildTargetValid() won't be checked.

It might also be worth changing the line

```c#
setter.Value.Invoke();
```

to 

```c#
Configure(setter.Key);
```

Reply:
i was discussing that with @keveleigh. i do not believe there are scenarios where filter will be null. however, since the method is public and the default value is null, i was reluctant to remove that code.

i WILL make the recommended change